builtin/settingtypes.txt

```diff
@@ -896,9 +896,15 @@ font_shadow (Font shadow) int 1
 #    Opaqueness (alpha) of the shadow behind the default font, between 0 and 255.
 font_shadow_alpha (Font shadow alpha) int 127 0 255
 
-#    Font size of the default font in point (pt).
+#    Font size of the default font where 1 unit = 1 pixel at 96 DPI
```

Comment:
Is this still correct when that change was undone?

Reply:
The reverted change was to 72 DPI, 96 is the original way.  I changed the docs to clarify that it actually isn't real points.